wrangler.example.toml

```diff
@@ -1,12 +1,9 @@
 name = "notion-cloudflare-worker"
-webpack_config = "webpack.config.js"
```

Comment:
https://developers.cloudflare.com/workers/wrangler-legacy/migration/eject-webpack/#i-use-type--webpack-and-webpack_config--pathtowebpackconfigjs-to-handle-jsx-typescript-webassembly-html-files-and-other-non-standard-filetypes